CMakeLists.txt

```diff
@@ -2,9 +2,25 @@ cmake_minimum_required(VERSION 2.8.12)
 
 project(OdbCMake)
 
+option (AT_ONCE "Build all structures into one file." FALSE)
+
+set(TARGET_DB "sqlite")
+
+# windows section - generate paths based on ODB_ROOT 
+if(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
+	if(DEFINED ENV{ODB_ROOT})
+		set(ODB_ROOT $ENV{ODB_ROOT})
+	endif(DEFINED ENV{ODB_ROOT})
+    message(STATUS "Prepare variables using ODB_ROOT=${ODB_ROOT}")
+	set(PC_LIBODB_INCLUDE_DIRS "${ODB_ROOT}\\include")
+	set(PC_LIBODB_LIBRARY_DIRS "${ODB_ROOT}\\lib")
+	set(ODB_LIBODB_INCLUDE_DIRS "${ODB_ROOT}\\include")
+	set(ODB_LIBRARY_PATH "${ODB_ROOT}\\lib")
+endif(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
```

Comment:
Just endif(), not done anywhere else.
This can also be entirely replaced by simply passing CMAKE_PREFIX_PATH on the command line, pointing to what you called ODB_ROOT